node/typings/Checkout.ts

```diff
@@ -232,7 +232,7 @@ interface PayloadItem {
 }
 
 interface SimulationPayload {
-  country: string
+  country?: string
```

Comment:
Is this change safe?

Reply:
Yes!! the simulation API doesn't require this field.